src/plugins/fs/generic.c

```diff
@@ -51,6 +51,84 @@ typedef enum {
     BD_FS_GET_FREE_SPACE,
 } BDFsOpType;
 
+    BDFSFsckFlags fsck;
+    BDFSConfigureFlags configure;
+    BDFSFeatureFlags features;
```

Comment:
What is this? A copy pasta?

Reply:
I'm surprised it compiled. It was probably just some random accidental Ctrl+C/Ctrl+V. Fixed.